docs/section21/Rule21-5.md

```diff
@@ -0,0 +1,70 @@
+
+# Boiler - Rule 21-5  
+
+**Rule ID:** 21-5  
+**Rule Description:** The baseline building design boiler plant shall be modeled as having a single boiler if the baseline building design plant serves a conditioned floor area of 15,000sq.ft. or less, and as having two equally sized boilers for plants serving more than 15,000sq.ft.  
+**Rule Assertion:** B-RMR  
+**Appendix G Section:** Section 21 Boiler  
+**Appendix G Section Reference:** Section G3.1.3.2 Building System-Specific Modeling Requirements for the Baseline model  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-1, 5, 7, 11, or 12
+2. B-RMR is not modeled with purchase heating
+
+**Manual Check:** None  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+**Function Call:**
+
+  1. get_zone_conditioning_category()
+
+**Applicability Check:**
+
+1. B-RMR is modeled with at least one air-side system that is Type-1, 5, 7, 11, or 12: `if PLACEHOLDER_AIRSIDE_SYSTEM_RULE-X-XX:`
+2. B-RMR is not modeled with purchased heating: `if NOT Rule-21-1:`
+
+## Rule Logic:  
+
+- Get zone conditioning category dictionary for B_RMR: `zcc_b = get_zone_conditioning_category(B_RMR)`
+
+- For each zone in B_RMR: `for zone_b in B_RMR...zones:`
+
+  - Check if zone has HVAC terminals, get zone terminals: `if zone_b.terminals: zone_terminals_b = zone_b.terminals`
+
+    - For each zone terminal: `for zone_terminal_b in zone_terminals_b:`
+
+      - Check if zone area has not been added to any heating hot water loop: `if NOT zone_flag:`
+
+        - Get HVAC system serving the terminal: `hvac_b = zone_b.terminals.served_by_heating_ventilation_air_conditioning_systems`
+
+          - Get heating system from HVAC system: `heating_system_b = hvac_b.heating_system`
```

Comment:
@xingcx as mentioned on another RDS, the zone > terminal connection has not been finalized and this might have to be changed in the future.

Reply:
So I think the logic should be as follows:
1. Identified all zones that are conditioned based on get_zone_conditioning_category(B_RMR) (List 1)_
2. Identify all zones that are EITHER (a) connect to HW fluid loop through Terminal or (b) through HeatingVentilationAirConditioningSystem -> HeatingSystem. (List 2) I don't think you do (a)?
3. Calculate the total area of all zones that are included in both List 1 and List 2

Once the terminal connections are finalized, we should revisit to confirm the logic.

Reply:
other than (1), i believe the RDS is doing everything else on the list.

Reply:
@supriyagoel I agree, don't see where zcc_b is used - the RDS just seems to be doing (2).
Another issue is that my logic above does not account for indirectly-conditioned spaces which may be included in List 1 but not List 2 - e.g., if an indirectly conditioned space is not by any heating system. I think area of such zones should also be counted.

Reply:
@mkarpman correct, I think Xing pointed that out as well somewhere. Not sure how that would work though - we need a data element to identify if the indirectly conditioned zone is being conditioned by a zone that falls under (2). Zones have a transfer air flow rate but no data element that would identify where the transfer air is coming from. Any thoughts?

Reply:
Few thoughts:
1. I think we agreed to have a function that calculates conditioned floor area. (This is also mentioned in 21-10.) So I would include a call to this future function here and in 21-10, and move the logic for determining the conditioned floor area that we started developing here to that function. To support the functionality that we need, perhaps the function should also or alternatively return the list of conditioned zones (List 1 per my comment from Nov 16 above).
2. To your question about indirectly conditioned zones, I would include area of zones that do not have terminals but that are included in the list of conditioned zones produced by the function discussed in #1. I think this will be correct in most cases and will be a conservative assumption as will result in more stringent requirements. The RDS may generate a message with the list of zones that have no terminals that are included in the conditioned area total, and the modeler can then explain to reviewer if the calculation was off. (If it is off.)

Reply:
@mkarpman that makes sense. I think having the list of conditioned zones which do not have terminals will be a good indication of an indirectly conditioned space.